### ISSUE TITLE
Source code update

### ISSUE THREAD

**Issue by pejakm** [heruoxin/Clip-Stack#49]
Hi.
PlayStore version of your app is at 1.7.2, while the code here is only at 1.5.0 and haven't been touched for 8 months. Are you planning to update anytime soon?

**Comment by DoubleTime** [comment]
Yeah I will like to take a view on Swipe Listener, the swipe delete animation is different with the PlayStore version.

In this git version when swipe to delete and I swipe again it still return to delete mode.

**Comment by Primokorn** [comment]
Dump...
